src/js/components/Modal/Modal.stories.tsx

```diff
@@ -161,101 +162,140 @@ export const Fullscreen = () => {
   return (
     <>
       <Button onClick={() => setIsOpen(true)}>Test Modal</Button>
-
       <Modal
         heading="Full bleed"
+        customClose={<Button size='large'>Save and Exit</Button>}
```

Comment:
thoughts on just making this property `closeButton` instead of `customClose`?

Reply:
Sure, that makes sense!

Reply:
@saralohr let me know if you are going to make the change or if you want me to make it

Reply:
Thanks, I think I can make it @BillyFigueroa though I may hit you up with some questions.